scripts/test_cover.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 # Define the directories to exclude
-EXCLUDE_DIRS=("/malicious")
+EXCLUDE_DIRS=("/test/util")
```

Comment:
A suggestion about the PR title: This PR does more than "removing race detection", it also "excludes the "/test/util" directory from code coverage analysis".  You may consider updating the title.